### PR TITLE
ci: adds a kib bump to CAPPP

### PR DESCRIPTION
**Pull request by faiq**
**What problem does this PR solve?**:
Whenever KIB releases we should bump CAPPP as well

**Which issue(s) does this PR fix?**:

* https://jira.d2iq.com/browse/D2IQ-NUMBER
* 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: .github/workflows/release-main.yml [hunk 1]
```diff
@@ -47,3 +47,39 @@ jobs:
...
+    needs: release-please
+    steps:
+      - if: ${{ needs.release-please.outputs.release_created }}
+        name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+           go-version: '1.17.0'
+
+      - if: ${{ needs.release-please.outputs.release_created }}
+        name: Fetch Konvoy
```
Comment (dkoshkin) [inline .github/workflows/release-main.yml:62]: typo?
```suggestion
        name: Fetch CAPPP
```

File: .github/workflows/release-main.yml [hunk 2]
```diff
@@ -47,3 +47,39 @@ jobs:
...
+        run: git config --global --add safe.directory "$GITHUB_WORKSPACE"
+
+      - if: ${{ needs.release-please.outputs.release_created }}
+        name: Update KIB in CAPPP
+        run: make update-kib
+        env:
+           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - if: ${{ needs.release-please.outputs.release_created }}
+        name: Create Pull Request for CAPPP bump in konvoy2
```
Comment (dkoshkin) [inline .github/workflows/release-main.yml:79]:
```suggestion
        name: Create Pull Request for CAPPP bump in CAPPP
```

**Review by dkoshkin — COMMENTED** [review]
Is it possible for the CAPPP repo to watch this repo instead?
I found https://github.community/t/triggering-by-other-repository/16163/12 and https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#repository_dispatch

Having the actions here forces KIB to "know" details about CAPPP and how to make changes.

File: .github/workflows/release-main.yml (hunk 1 above)
Comment (faiq) [reply .github/workflows/release-main.yml]: Very poor copy pasting skills. Sorry :bow:

**Pull request merged**